src/commands/firestore-databases-create.ts

```diff
@@ -63,13 +67,21 @@ export const command = new Command("firestore:databases:create <database>")
         ? types.PointInTimeRecoveryEnablement.ENABLED
         : types.PointInTimeRecoveryEnablement.DISABLED;
 
+    let cmekConfig: types.CmekConfig | undefined;
+    if (options.kmsKeyName) {
+      cmekConfig = {
+        kmsKeyName: options.kmsKeyName,
+      };
+    }
+
     const databaseResp: types.DatabaseResp = await api.createDatabase(
       options.project,
       database,
       options.location,
       type,
       deleteProtectionState,
       pointInTimeRecoveryEnablement,
+      cmekConfig,
```

Comment:
Optional: Create database has 7 args - consider switching this to a parameter object (ie https://go/tott/550)

Reply:
Have switched to a param object, thanks